Source/WebKit/UIProcess/WebAuthentication/AuthenticatorManager.cpp

```diff
@@ -371,6 +371,27 @@ void AuthenticatorManager::requestPin(uint64_t retries, CompletionHandler<void(c
     });
 }
 
+void AuthenticatorManager::requestNewPin(uint64_t minLength, CompletionHandler<void(const WTF::String&)>&& completionHandler)
+{
+    auto callback = [weakThis = WeakPtr { *this }, this, completionHandler = WTFMove(completionHandler)] (const WTF::String& pin) mutable {
```

Comment:
This doesn't comply with safer C++ guidelines. Please do not capture `this`, only `weakThis`